public/src/pages/LoginPage.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+
+const LoginPage = () => {
+    return (
+        <div style={{
+            display: "flex",
+            justifyContent: "center",
+            alignItems: "center",
+            height: "100vh",
```

Comment:
Please use the `style.scss` to define styles (or tell me why having them inline is better)